collector.go

```diff
@@ -277,6 +288,47 @@ func collectPerSubscriptionMetric(stats *stats, desc *prometheus.Desc, collectFu
 	})
 }
 
+func collectParkedMessagesPerSubscriptionMetric(stats *stats, desc *prometheus.Desc, ch chan<- prometheus.Metric) {
+	jp.ArrayEach(stats.subscriptionsStats, func(jsonValue []byte, dataType jp.ValueType, offset int, err error) {
+		eventStreamID, _ := jp.GetString(jsonValue, "eventStreamId")
+		groupName, _ := jp.GetString(jsonValue, "groupName")
+
+		// get parked messages max event number
+		parkedMessagesURL := fmt.Sprintf("/streams/$persistentsubscription-%s::%s-parked/head/backward/1", eventStreamID, groupName)
```

Comment:
The code for getting statistics out of ES should be kept in `eventstore_client`, please refactor.